proctracer.json.sample

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
As the extension suggests this is just a sample file to help users with the plugin configuration. I can delete it, or  move it elsewhere (inside the plugin directory?) if you want.